trySwift/SpeakerDetailViewController.swift

```diff
@@ -11,7 +11,7 @@ import TrySwiftData
 
 class SpeakerDetailViewController: UITableViewController {
 
-    var speaker: Speaker!
```

Comment:
Should I go ahead and show an empty VC at the start? Other than that, I think this is good to go now.